integrations/azure-devops/azure_devops/client/azure_devops_client.py

```diff
@@ -28,13 +28,28 @@ def create_from_ocean_config(cls) -> "AzureDevopsClient":
         event.attributes["azure_devops_client"] = azure_devops_client
         return azure_devops_client
 
+    async def get_single_project(self, project_id: str) -> dict[str, Any]:
+        project_url = (
+            f"{self._organization_base_url}/{API_URL_PREFIX}/projects/{project_id}"
+        )
+        project = (await self.send_request("GET", project_url)).json()
+        return project
+
     @cache_iterator_result()
-    async def generate_projects(self) -> AsyncGenerator[list[dict[str, Any]], None]:
+    async def generate_projects(
```

Comment:
Add a comment explaining the sync_default_team scenario
Explain why does it exist and add links to ADO's docs